packages/xo-web/src/common/xo/index.js

```diff
@@ -1018,6 +1056,7 @@ export const startVms = vms =>
       return error(_('failedVmsErrorTitle'), _('failedVmsErrorMessage', { nVms: nErrors }))
     }
 
+    // Clone or retry with force
```

Comment:
We should probably ignore the MAC address check in this case.

Reply:
Yes, it needs to be ignored, otherwise the VMs that already failed with both errors will fail again because of duplicated MAC addresses.